pubsublite/streaming-analytics/README.md

```diff
@@ -0,0 +1,107 @@
+# Pub/Sub Lite with Cloud Dataflow
+
+[![Open in Cloud Shell](http://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/java-docs-samples&page=editor&open_in_editor=pubsublite/streaming-analytics/README.md)
+
+Samples showing how to use [Pub/Sub Lite] with [Cloud Dataflow].
+
+## Pub/Sub Lite to Cloud Storage sample
+
+[PubsubliteToGcs.java](examples/PubsubliteToGcs.java)
+
+This sample shows how to create an [Apache Beam] streaming pipeline that reads
+messages from [Pub/Sub Lite], group the messages using a fixed-sized windowing
+function, and writes them to [Cloud Storage].
+
+Resources needed for this example:
+
+1. A pair of Pub/Sub Lite topic and subscription. 
+2. A Cloud Storage bucket.
+
+### Setting up
+
+1. [Enable the APIs](https://console.cloud.google.com/flows/enableapi?apiid=dataflow,compute_component,logging,storage_api,pubsublite.googleapis.com): Cloud Dataflow, Compute Engine, Cloud Logging, Cloud Storage, Pub/Sub Lite.
+
+1. Create a Cloud Storage bucket.
+
+   ```bash
+   PROJECT=$(gcloud config get-value project)
+   BUCKET=your-gcs-bucket
```

Comment:
nit: add note that bucket name has to be globally unique

Reply:
Sometimes it's nice to set it up for them with their project id ie `BUCKET="$PROJECT-pubsublite-dataflow-bucket"